### PR TITLE
Update token steps for DataWarehouse API

### PR DESCRIPTION
**Pull request by tanyalai**
Updated and reordered instructions to use the latest version of Postman. Added updated instructions image.

The current instructions don't correspond to the current Postman version and is generating customer calls and ICMs.

### CODE REVIEW

File: memdocs/intune/developer/reports-proc-data-rest.md [hunk 1]
```diff
@@ -110,35 +110,36 @@ For more information, see [Intune Data Warehouse API endpoint](reports-api-url.m
 
 To get a new access token for Postman, you must add the Azure AD authorization URL, add your Client ID, and Client Secret. Postman will load the authorization page where you will type your credentials.
 
+Before you make the call, verify that you have already added the Callback URL to your app in Azure. The Callback URL is `https://www.getpostman.com/oauth2/callback`
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/reports-proc-data-rest.md:113]:
```suggestion
Before you make the call, verify that you have already added the Callback URL to your app in Azure. The Callback URL is `https://www.getpostman.com/oauth2/callback`.
```

File: memdocs/intune/developer/reports-proc-data-rest.md [hunk 2]
```diff
@@ -110,35 +110,36 @@ For more information, see [Intune Data Warehouse API endpoint](reports-api-url.m
...
     `https://fef.tenant.manage.microsoft.com/ReportingService/DataWarehouseFEService/dates?api-version=v1.0`
 4. Choose the **Authorization** tab, and select **OAuth 2.0** from the **Type** list.
-5. Select **Get New Access Token**.
-6. Verify that you have already added the Callback URL to your app in Azure. The Callback URL is `https://www.getpostman.com/oauth2/callback`.
-7. Type Bearer for the **Token Name**.
-8. Add the **Auth URL**. It should look something like:  
+5. Scroll down to the **Configure New Token** section.
+6. Type Bearer for the **Token Name**.
+7. Select **Authorization Code** as the Grant Type.
+8. Add the **Callback URL**. The callback url is `https://www.getpostman.com/oauth2/callback`
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/reports-proc-data-rest.md:126]:
```suggestion
8. Add the **Callback URL**. The callback URL is `https://www.getpostman.com/oauth2/callback`.
```

File: memdocs/intune/developer/reports-proc-data-rest.md [hunk 3]
```diff
@@ -156,7 +157,7 @@ To get a new access token for Postman, you must add the Azure AD authorization U
 
 The following sample contains a simple REST client. The code uses the **httpClient** class from the .NET library. Once the client gains credentials to Azure AD, the client constructs a GET REST call to retrieve the dates entity from the Data Warehouse API.
 
-> [!Note]  
+> [!Note]
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/reports-proc-data-rest.md:160]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tanyalai, kindly commit the minor suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**